tests/test_wps_subset.py

```diff
@@ -0,0 +1,282 @@
+
+        # let's start with one example...
+        resource = os.path.join(
+            config_dict['opendap_path'],
+            'pairing_day_global-reg-grid_360_720_nobounds_ref180.nc')
+
+        execution = test_wps_utils.execute(
+            'subset', inputs=[
+                ('resource', resource),
+                ('typename', 'testgeom:montreal_circles'),
+                ('featureids', 'montreal_circles.43'),
+                ('geoserver', config_dict['geoserver'])],
+            wps_host=config_dict['wps_host'], wps_client=self.client)
+
+        while execution.status == 'ProcessAccepted':
+            execution.checkStatus(sleepSecs=1)
+        output_json = execution.processOutputs[0].reference
+        if output_json[:7] == 'file://':
+            output_json = output_json[7:]
+            f1 = open(output_json, 'r')
+            json_data = json.loads(f1.read())
+            f1.close()
+        else:
+            json_data = json.loads(test_wps_utils.get_wps_xlink(output_json))
+        output_netcdf = json_data[0]
+        if output_netcdf[:7] == 'file://':
+            tmp_output_netcdf = output_netcdf[7:]
+        else:
+            tmp_output_netcdf = '/tmp/testtmp.nc'
+            f1 = open(tmp_output_netcdf, 'w')
+            f1.write(test_wps_utils.get_wps_xlink(output_netcdf))
+            f1.close()
+
+        nc = netCDF4.Dataset(tmp_output_netcdf, 'r')
+        nclon = nc.variables['lon']
+        nclat = nc.variables['lat']
+        ncvar = nc.variables['pairing']
+        self.assertEqual(nclon.shape, (21,))
+        self.assertEqual(nclat.shape, (21,))
+        self.assertEqual(ncvar.shape, (365, 21, 21))
+        self.assertTrue(ncvar[0, 0, 0] is ma.masked)
+        self.assertEqual(ncvar[0, 10, 10], 271213.0)
+        self.assertEqual(nc.subset_typename, 'testgeom:montreal_circles')
+        self.assertEqual(nc.subset_featureid, 'montreal_circles.43')
+        nc.close()
+
+    def test_subset_wfs_opendap_01_default_geoserver(self):
+        # pairing0.1.1-montreal_circles0.1.1
+        config_dict = test_wps_utils.config_is_available(
+            'pairing0.1.1-montreal_circles0.1.1',
+            ['opendap_path', 'fileserver_path', 'test_default_geoserver'],
+            self.config, set_wps_host=True)
+
+        # let's start with one example...
+        resource = os.path.join(
+            config_dict['opendap_path'],
+            'pairing_day_global-reg-grid_360_720_nobounds_ref180.nc')
+
+        execution = test_wps_utils.execute(
+            'subset', inputs=[
+                ('resource', resource),
+                ('typename', 'testgeom:montreal_circles'),
+                ('featureids', 'montreal_circles.43')],
+            wps_host=config_dict['wps_host'], wps_client=self.client)
+
+        while execution.status == 'ProcessAccepted':
+            execution.checkStatus(sleepSecs=1)
+        output_json = execution.processOutputs[0].reference
+        if output_json[:7] == 'file://':
```

Comment:
why is `if` needed? In a test you have a fixed expectation.

Reply:
@huard That's an Oranus process. I leave it to you to change it correctly